src/app/MarineApp/components/MetricsContext/MetricsContext.tsx

```diff
@@ -0,0 +1,48 @@
+import { createContext, FC, useCallback, useContext, useEffect, useMemo, useState } from "react"
+
+type notifyParams = {
+  widgetName: string
+  visible: boolean
+}
+
+type MetricsContextValue = {
+  elementsVisible?: boolean
+  notifyVisibility?: (arg: notifyParams) => void
```

Comment:
The argument is called `arg` here, but `element` in the implementation. Would be nice if it were consistent.